src/main/java/homework_2/random_chars_table/RandomCharsTable.java

```diff
@@ -0,0 +1,75 @@
+package homework_2.random_chars_table;
+
+import homework_2.utils.App;
+
+import java.util.Random;
+
+public class RandomCharsTable extends App {
```

Comment:
Somehow, without letters it removes the line
![image](https://user-images.githubusercontent.com/58872509/126624149-ae122cfc-2426-4a94-bef8-1eed4c99c273.png)

Reply:
Good job! All tests are passed, works as expected. For style, it's good that you tried most of the things we learn, but in prod, I don't recommend doing it, as simplicity is usually the main goal

Reply:
Opt: make homework simpler to undertand where possible

Reply:
If there are no letters suitable for the strategy, then the string - "* Strategy * letters" - should not be displayed? Did I understand correctly?
Of course, I added inheritance for the sake of inheritance. In real tasks, this is not necessary, I understand that.